astroquery/vo_conesearch/conesearch.py

```diff
@@ -78,21 +74,33 @@ class AsyncConeSearch(AsyncBase):
     If no ``timeout`` keyword given, it waits until completion:
 
     >>> async_result = async_search.get(timeout=30)
-    >>> cone_arr = async_result.array.data
-    >>> cone_arr.size
-    36184
+    >>> len(async_result)
+    74271
 
     """
     def __init__(self, *args, **kwargs):
         super(AsyncConeSearch, self).__init__(conesearch, *args, **kwargs)
 
 
-def conesearch(center, radius, verb=1, catalog_db=None, pedantic=None,
-               verbose=True, cache=True, timeout=None, query_all=False):
+def conesearch(center, radius, verb=1, catalog_db=None,
```

Comment:
I wonder whether we can take these kwargs as keyword only arguments, or that would upset downstream too much? At least that would make sure that the removal of `pedantic` etc doesn't hide in their code and cause cryptic issues.

Reply:
Yeah, fine by me. Done.